django/contrib/sessions/backends/base.py

```diff
@@ -147,24 +156,60 @@ def is_empty(self):
         except AttributeError:
             return True
 
+    def _get_session_key_part(self, session_key):
+        "Return a string of the session_key key."
+        try:
+            algorithm, key = session_key.split(SESSION_KEY_DELIMITER)
+            if algorithm == SESSION_HASHING_ALGORITHM:
+                return key
+            return None
+        except ValueError:
+            return None
+
+    def _is_hashed(self, session_key):
+        "Return True when the session_key is hashed in the backend."
+        try:
+            return bool(self._get_session_key_part(session_key))
+        except IndexError:
```

Comment:
It's weird to handle `ValueError` in `_get_session_key_part` and `IndexError` here...

Simplify to:

```python
return session_key.startswith(SESSION_HASHED_KEY_PREFIX)
```